### PR TITLE
Run more tests in CI

### PR DESCRIPTION
**Pull request by akalin-keybase**
Also reduce a few timeouts to 30s.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -279,7 +279,40 @@ def runNixTest(prefix) {
 
     sh 'go install github.com/keybase/kbfs/...'
 
+    // Keep the list below in sync with the result of
+    //   find . \( -path ./vendor -o -path ./.git \) -prune -o -type d | sort
+
     tests = [:]
+    tests[prefix+'cache'] = {
```
Comment (akalin-keybase) [inline Jenkinsfile]: Would be nice if there were a more concise way to do this

**Comment by akalin-keybase** [comment]
kicking over to jzila since strib's out

**Comment by jzila** [comment]
I'm out too, but I can take a look later today.

**Comment by akalin-keybase** [comment]
my mistake, kicking over to @songgao!

File: appveyor.yml [hunk 2]
```diff
@@ -51,6 +51,7 @@ build_script:
   - cd dokan
   - go test -i
   - go test -c
+  - dokan.test
```
Comment (akalin-keybase) [inline appveyor.yml]: looks like we were building the dokan test but not running? @zanderz was there a reason?

**Comment by jzila** [comment]
Please don't merge until I take a look later today.

**Pull request merged**